data/pigui/mainmenu.lua

```diff
@@ -0,0 +1,213 @@
+-- Copyright © 2008-2018 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Engine = import('Engine')
+local Game = import('Game')
+local Ship = import("Ship")
+local ShipDef = import("ShipDef")
+local SystemPath = import("SystemPath")
+local Equipment = import("Equipment")
+local Format = import("Format")
+local ui = import('pigui/pigui.lua')
+local Vector = import('Vector')
+local Event = import('Event')
+local Lang = import("Lang")
+local lc = Lang.GetResource("core")
+local lui = Lang.GetResource("ui-core")
+local qlc = Lang.GetResource("quitconfirmation-core")
+local elc = Lang.GetResource("equipment-core")
+local clc = Lang.GetResource("commodity")
+
+local cargo = Equipment.cargo
+local misc = Equipment.misc
+local laser = Equipment.laser
+local hyperspace = Equipment.hyperspace
+
+local player = nil
+local colors = ui.theme.colors
+local pionillium = ui.fonts.pionillium
+local icons = ui.theme.icons
+
+local mainButtonSize = Vector(400,46) * (ui.screenHeight / 1200)
+local dialogButtonSize = Vector(150,46) * (ui.screenHeight / 1200)
+local mainButtonFontSize = 36 * (ui.screenHeight / 1200)
+
+local showQuitConfirm = false
+local quitConfirmMsg
+local max_flavours = 19
+
+local startLocations = {
+	{['name']=lui.START_AT_EARTH,
+	 ['desc']=lc.START_AT_EARTH_DESC,	 
+	 ['location']=SystemPath.New(0,0,0,0,6,48600),
+	 ['shipType']='sinonatrix',['money']=100,hyperdrive=true,
+	 ['equipment']={
+		{laser.pulsecannon_1mw,1},
+		{misc.atmospheric_shielding,1},
+		{misc.autopilot,1},
+		{misc.radar,1},
+		{cargo.hydrogen,2}}},
+	{['name']=lui.START_AT_NEW_HOPE,
+	 ['desc']=lc.START_AT_NEW_HOPE_DESC,
+	 ['location']=SystemPath.New(1,-1,-1,0,4),
+	 ['shipType']='pumpkinseed',['money']=100,hyperdrive=true,
+	 ['equipment']={
+		{laser.pulsecannon_1mw,1},
+		{misc.atmospheric_shielding,1},
+		{misc.autopilot,1},
+		{misc.radar,1},
+		{cargo.hydrogen,2}}},
+	{['name']=lui.START_AT_BARNARDS_STAR,
+	 ['desc']=lc.START_AT_BARNARDS_STAR_DESC,
+	 ['location']=SystemPath.New(-1,0,0,0,16),
+	 ['shipType']='xylophis',['money']=100,hyperdrive=false,
+	 ['equipment']={
+		{misc.atmospheric_shielding,1},
+		{misc.autopilot,1},
+		{misc.radar,1},
+		{cargo.hydrogen,2}}}
+}
+
+local function dialogTextButton(label, enabled, callback)
+	local bgcolor = enabled and colors.buttonBlue or colors.grey
```

Comment:
indentation

Reply:
This should be indented...it's a function...or am I missing something?

Reply:
From remarks in chat, this sounds like it's tabs vs spaces. I'll take a look.

Reply:
This should be better now...there was a mix of spaces and tabs in these functions, I converted to all tabs.